system/lib/pthread/emscripten_futex_wake.c

```diff
@@ -11,7 +11,7 @@
 #include <atomic.h>
 #include <emscripten/threading.h>
 
-int _emscripten_thread_supports_atomics_wait(void);
+int emscripten_is_main_browser_thread(void);
```

Comment:
This function is declared in  <emscripten/threading.h> so I think you can remove this line.

Reply:
check